meta/documents/user_guide/en/plugin_installation.md

```diff
@@ -0,0 +1,45 @@
+# Wayfair plugin: Installation instructions
+
+1. Login to the Plentymarkets system
+
+2. Go to **plugins >> plentyMarketplace**, which will use the system owner’s credentials to log you in on the Marketplace
+
+3. Click the search button  in the top-right corner of the page
+
+4. Enter “Wayfair” into the box and press enter
+
+5. If “Already purchased” appears on the page, skip to step 8
+
+6. Click the **go to checkout** button
+7. Complete the purchasing form
+    1. Accept terms and conditions
+    2. Click **order now**
+
+8. Return to the Plentymarkets system, and go to **Plugins >> Plugin Overview**.

```

Comment:
This has the deprecated tag but I remember the new way was very confusing and didn't work as expected. Did they fix the new way or say when this would be totally removed?

Reply:
I meant to update this to using the new one - maybe I messed up.

Reply:
Ah crap this is the old document. Great catch.

Reply:
@mcloughlin77 new doc is up.